nsinit/exec.go

```diff
@@ -51,7 +45,7 @@ func execAction(context *cli.Context) {
 	}
 	process := &libcontainer.Process{
 		Args:   context.Args(),
-		Env:    context.StringSlice("env"),
+		Env:    append(os.Environ(), context.StringSlice("env")...),
```

Comment:
Oh ya, this is for when you are running nsinit from a process supervisor the env comes from what you setup in a unit file or upstart script